src/app/@ansyn/ansyn/package.json

```diff
@@ -1,22 +1,21 @@
 {
 	"name": "@ansyn/ansyn",
-	"version": "2.1.2",
+	"version": "3.0.0",
 	"peerDependencies": {
 		"@ngrx/core": "^1.2.0",
 		"@ngrx/store": "^7.2.0",
 		"@ngrx/effects": "^7.2.0",
 		"@angular/common": "^7.2.2",
 		"@angular/core": "^7.2.2",
-		"@ansyn/imagery": "2.1.2",
-		"@ansyn/map-facade": "2.1.2",
-		"@ansyn/menu": "2.1.2",
-		"@ansyn/imagery-cesium": "2.1.2",
-		"@ansyn/imagery-video": "2.1.2",
+		"@ansyn/imagery": "3.0.0",
+		"@ansyn/map-facade": "3.0.0",
+		"@ansyn/menu": "3.0.0",
+		"@ansyn/ol": "3.0.0",
+		"@ansyn/imagery-cesium": "3.0.0",
+		"@ansyn/imagery-video": "3.0.0",
 		"@ansyn/event-drops": "^0.2.8-1.9",
-		"ol": "^5.3.0",
 		"@angular/cdk": "^7.3.0",
-		"@angular/material": "^7.3.0",
-		"@ngx-translate/core": "^11.0.1"
```

Comment:
Why did this package change from  peer dependency to dependency?

Reply:
because it more suitable to be dependency.
why should i force our client to run `npm i @ngx-translate/core`